src/components/ProductListContainer/index.js

```diff
@@ -41,7 +41,8 @@ const ProductListContainer = ({ productIds, featuredProductId }) => {
             suscriptorsNumber={suscriptorsNumber}
             expirationDate={expirationDate}
             discountPercentage={discountPercentage}
-            extended={id === featuredProductId}
+            extended={false}
```

Comment:
This is a hardcoded `extended`, I think you mentioned it in the call.

There are multiple ways of applying styles with boolean condition, the one I like most is using `classnames` library. You can do something like this
```
<div className={classnames({
  [classes.extendedCard]: extended,
  [classes.card]: !extended
})} />
```

So you apply the cardExtended class when `extended` is true, otherwise you apply the normal card class.